posthog/batch_exports/http.py

```diff
@@ -526,6 +526,36 @@ def perform_destroy(self, instance: BatchExport):
         """
         disable_and_delete_export(instance)
 
+    @action(methods=["POST"], detail=True)
+    def test(self, request: request.Request, *args, **kwargs) -> response.Response:
```

Comment:
is this endpoint just for fetching a test (and it's list of steps) for a given batch export?

Reply:
Yeah, thought about having everything in a single endpoint, but it looked too complicated. I'd be happy to rename this to something more verbose though...

Reply:
I think it does make sense to get a list of possible tests upfront, especially if we want to add them to destinations one at a time, since the frontend could probably not even show a test button if there are no tests to run for example. Was also wondering if it could just be a GET method if it's not actually running anything